### ISSUE TITLE
Using opposite relation for Offer entity

### ISSUE THREAD

**Issue by jkabat** [schemaorg/schemaorg#4306]
There is clearly described the way how to link Offers with Product or Service.
However for different type of business I need to create link between Person/Organization and Offer.

I know its possible using `makesOffer` property. But can we use it as a list of multiple offers?
```json
{
  "@context": "http://schema.org/",
  "@type": "Organisation",
  "name": "My Org"
  "makesOffer": [
    {
      "@type": "Offer",
      "@id": "http://my-domain.com/...",
      "itemOffered": {...}
      "priceSpecification": {...}
    },
    {
      "@type": "Offer",
      "@id": "http://my-domain.com/...",
      "itemOffered": {...}
      "priceSpecification": {...}
    }
  ]
}
```

I might use `offerCatalog` property but I dont know if its supported by google rich snippets. I want to benefit from defined structure as much as possible...

**Comment by jvandriel** [comment]
As far as I'm aware Google does not support the use of the `makesOffer` property nor the `OfferCatalog` class. However, to make sure you'll have to go through [Google's developers documentation](https://developers.google.com/search/docs/appearance/structured-data/search-gallery).

Here we focus on the development of the vocabulary itself and rarely discuss any search features by any of the search engines.

**Comment by jkabat** [comment]
You are right @jvandriel, my question was tightly related to the use of search engine. I was just so focused on searching the answers in this repository and its issues, that it did not strike me...

Thank you for an answer though.
